genetIC/src/simulation/modifications/linearmodification.hpp

```diff
@@ -178,10 +178,9 @@ namespace modifications {
   template<typename DataType, typename T=tools::datatypes::strip_complex<DataType>>
   class VelocityModification : public OverdensityModification<DataType, T> {
   private:
-    const bool derivInRealSpace = true;
-
+    bool derivInRealSpace = true; //!< If true, make the derivation in real space. Otherwise, make the derivation in Fourier space.
```

Comment:
Do you mean here `make the derivative in Fourier space` rather than `make the derivation in Fourier space`?